prisma/schema.prisma

```diff
@@ -7,27 +7,38 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum CommentStatus {
+  pending
+  approved
+  rejected
+}
+
 model Comment {
-  id          Int      @id @default(autoincrement())
-  accidentId  String   @map("accident_id") @db.VarChar
-  text        String
-  authorId    String   @map("author_id") @db.VarChar
-  isPublished Boolean  @default(false) @map("is_published")
-  createdAt   DateTime @default(now()) @map("created_at")
-  updatedAt   DateTime @default(now()) @map("updated_at")
-  author      User     @relation(fields: [authorId], references: [id])
+  id         Int    @id @default(autoincrement())
+  accidentId String @map("accident_id") @db.VarChar
+  text       String
+
+  authorId String @map("author_id") @db.VarChar
+  author   User   @relation(fields: [authorId], references: [id])
+
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @default(now()) @map("updated_at")
+
+  status CommentStatus @default(pending)
```

Comment:
`prettier-plugin-prisma` (#57) automatically aligns columns, so pretty much any change creates a 10-line diff.  I’ve split attributes into groups to reduce these diffs in the future.

Reply:
Hm, why its not using fixed width columns as the original prisma formatter by default. Interesting this configurable...

Reply:
Maybe column width are calculated per group (which are separated by blank lines). I guess Prettier plugin for Prisma just wraps the standard formatter. Anyway, we can check options in other PRs. Ideally, we want diffs to be minimal to have a nice git blame. But the file is quite small to pay to much attention to this probably.